### DISCUSSION TITLE
Driver for EG start usb encoder

### DISCUSSION THREAD

**Discussion by stephaneweg** [rsta2/circle#444] [Ideas]
I am building an arcade machine. And want to develop my games without any OS on it

Is it possible for somebody to make à driver for this device?

https://www.rigacci.org/wiki/doku.php/doc/appunti/hardware/gamepad_dragonrise_eg_starts

**Comment by rsta2** [comment]
I think, you do not need a special driver. Circle already has a driver for 
generic USB gamepads and it should work with this device. You can try it with 
sample/27-usbgamepad.
